Feature

mikethreels opened the pull request:
## Milestone: Hello world app

### when creating the project the following points have been added.
- [x] Created an HTML form
- [x] added styling.

fernandorpm commented on css/main.css:
```diff
@@ -325,3 +325,48 @@ li {
+  margin: 5px 0 0 5px;
+}
+
+#first_name{
+  width: 39%; 
+}
+
+#last_name{
+  width: 39%;
+}
+
+#email_address {
+  width: 80%;
+}
+
+textarea {
+  margin: 5px;
+  width: 80%;
+  border: none;
+  background-color: rgb(241, 233, 216);
+}
```
- [ ] **[OPTIONAL]** If you added the desired `width` for your form as suggested above, you can just change all of the `width: 80%` to `width: 100%`. So if you ever change the desired `width` on your `.form-container`, it will change in all of its children!

- [ ] **[OPTIONAL]** About `first_name` and `last_name`: Instead of setting very specific widths to these 2 properties, you could have nested these 2 `inputs` inside a `div` and change the `div` display to `flex` and add a `gap`:

```css
#div_name {
  width: 100%; /* The .form-container already gives you the desired size */
  display: flex;
  gap: 2%; 
}

#first_name,
#last_name {
  width: 50%;
}
```

fernandorpm commented on index.html:
```diff
@@ -289,6 +289,26 @@ <h2>About me</h2>
+              <div class="info">
+                <textarea name="Info" id="information" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
+              </div>
+              <input type="submit" class="submit_button">
```
- [ ] **[REQUIRED CHANGES]** This button (submit input, in your case) should contain the text: 'Get in Touch', in order for the user to know what they should expect from the button. You can use the `value` property to do it.

fernandorpm commented on css/main.css:
```diff
@@ -325,3 +325,48 @@ li {
+  display: flex;
+}
+
+.contact_form_header {
+  max-width: 20vw;
+  margin: 50px 30px 0 10vw;
+}
```
- [ ] **[REQUIRED CHANGES]** Try to use a `sans-serif` font-family on the `.contact_form_header`, so it can match the Figma Template. I recommend you use the same font as the rest of the page, to increase the users' confidence in the page. A design that isn't consistent can give the impression of something fake, or of low quality, and hard your/your clients' trustfulness!

fernandorpm reviewed (changes requested):
# :warning: Changes Required on this Pull Request! :warning:
Hello there!

You have done a great job so far, and are one step closer to becoming the best developer ever! :sunglasses:

<img src="https://i.imgur.com/vfjSVsr.gif" width="300px" />

## Required
> These changes HAVE TO be completed in order for your project to be approved. This means that the project doesn't attend to Microverse's Standard yet. I know that it can be frustrating, but they should be seen as part of your path to become an even better Developer!

- :x: Your project doesn't have linters implemented to them :x:. Please remember that the linters are essential in order to make your code looks clean, more professional, and easier to understand for other programmers. I got you, [here's the link](https://github.com/microverseinc/linters-config).

- Check the comments under review marked with the tag **[REQUIRED CHANGES]**.

> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Just keep in mind that Microverse has a standard for the project requirements, to guarantee that you will be able to become a great developer, and both you and I have to follow them!
___

## Optional
> These changes are NOT mandatory. But if you want to sky-rocket :rocket: your skills and your projects, go for it! :sunglasses:

- Check the comments below marked with the tag **[OPTIONAL]**.

___
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Cheers and Happy Coding! :heart:

> *As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).*

mikethreels commented:
## Milestone: Contact form [Updated]

### when updating the project the following points have been added.
- [x] Updated the HTML so the input takes the required amount of characters
- [x] Added Formspree so the form actually works
- [x] changed input type to email for the email input.
- [x] updated styling.

fernandorpm commented on index.html:
```diff
@@ -289,6 +289,26 @@ <h2>About me</h2>
+          <div class="form_container">
+            <form class="contact form" action="https://formspree.io/f/xeqvdkea" method="POST">
+              <div class="name">
+                <input type="text" name="Name" id="first_name" placeholder="First name" maxlength="30">
+                <input type="text" name="Surname" id="last_name" placeholder="Last name" maxlength="30">
+              </div>
+              <div class="email">
+                <input type="email" name="Email" id="email_address" placeholder="Email address">
+              </div>
+              <div class="info">
+                <textarea name="Info" id="information" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
```
- [ ] **[REQUIRED CHANGES]** As mentioned in the last review, it is important that all of the `inputs` and the `textarea` have the `required` HTML property, in order to make sure that the user won't forget any important information!

fernandorpm replied:
- [ ] **[REQUIRED CHANGES]** As stated in the last review, it is important that you add the `required` HTML property in the `input` and `textarea` of the forms, so the user doesn't forget to submit any information!

fernandorpm commented on css/main.css:
```diff
@@ -325,3 +325,49 @@ li {
+  margin: 50px 30px 0 10vw;
+}
+
+.form_container {
+  margin: 50px 0 0 0;
+  width: 40vw;
+}
```
- [ ] **[REQUIRED CHANGES]** As stated in the last review and in the project requirements, it is important that you implement the `display:flex` to your sections, to allow you to improve in flex styling. The recommended tutorial mentioned in the last review is a great starting point if you're having difficulties with it.

- [ ] **[OPTIONAL]** I would also like to mention that **_if_** you feel like you have some extra time, try to work on the optional changes as suggested. They will help you to better understand the flexbox behavior, and will also make your project look more professional for whoever checks it!

fernandorpm reviewed (changes requested):
[REDACTED]
This project is to be reviewed by a different Code reviewer

Ghiftee reviewed (changes requested):
Hi there @mikethreels  👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏

To highlight 🍾 🎉 

- Your form successfully submits user data.
- You have a descriptive PR summary

You've done well implementing some of the requested changes, these are great and you should be proud of yourself 👏🏽  but there are still some issues that aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
